Duplicati/Library/Utility/TempFile.cs

```diff
@@ -35,7 +35,7 @@ public class TempFile : IDisposable
         /// </summary>
         public static string APPLICATION_PREFIX = Utility.getEntryAssembly().FullName.Substring(0, 3).ToLower(System.Globalization.CultureInfo.InvariantCulture) + "-";
 
-        private string m_path;
+        private string? m_path;

```

Comment:
This is being set to null to avoid it being read after being disposed. It is better to make it `readonly` and add a `m_isDisposed` flag.